lib/utils/DevServerPlugin.js

```diff
@@ -153,24 +165,12 @@ class DevServerPlugin {
 
     // use a hook to add entries if available
     if (EntryPlugin) {
-      compiler.hooks.make.tapPromise('DevServerPlugin', (compilation) =>
-        Promise.all(
-          additionalEntries.map(
-            (entry) =>
-              new Promise((resolve, reject) => {
-                compilation.addEntry(
-                  compiler.context,
-                  EntryPlugin.createDependency(entry, {}),
-                  {}, // global entry
-                  (err) => {
-                    if (err) return reject(err);
-                    resolve();
-                  }
-                );
-              })
-          )
-        )
-      );
```

Comment:
The original intention was to allow modifying entries after they've been added.

Reply:
Why? Can you clarify?

Reply:
If we reference `additionalEntries` outside in the hook, they can be altered after hooks have added.

Reply:
But why we need it? I can't undestand